### PR TITLE
updates production download pages

### LINKED ISSUE #4063: Update production download and documentation pages
https://revenuedata.doi.gov/downloads/federal-production-by-month/

https://revenuedata.doi.gov/downloads/federal-production/

### PR DESCRIPTION
**Pull request by lpgoldstein**
Fixes #4063 

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/onrr/doi-extractives-data/production_downloads/)

Changes proposed in this pull request:

updates text in production by month and production by year download pages

### CODE REVIEW

**Comment by Maroyafaied** [comment]
These are out of place
image

**Comment by Maroyafaied** [comment]
not italic 
image

**Comment by Maroyafaied** [comment]
production needs to be on its own line 
image

**Comment by brentryanjohnson** [comment]
Since @lpgoldstein is out, I'll drop in and fix these, @Maroyafaied.

File: src/markdown/downloads/federal-production-by-month.md [hunk 1]
```diff
@@ -47,19 +47,19 @@ The sum of 12-month production totals for a given calendar or fiscal year may no
...
-_Land Category_ This field distinguishes between onshore and offshore revenue. Some revenues – such as civil penalties – are not tied to either onshore or offshore.
+_Land Category_ This field distinguishes between onshore and offshore production.
 
 _Commodity_ We have monthly data for oil, gas, and coal, since these are high-volume commodities that result in the most revenue. This field includes the units for each commodity in parentheses.
 
-- _Oil Prod Vol (bbl)_ Oil production is measured in barrels (bbl)
-- _Gas Prod Vol (Mcf)_ Gas (natural gas) is measured in thousand cubic feet (mcf)
-- _Coal Prod Vol (ton)_ Coal is measured in tons. A ton is 2000 pounds.
+  - _Oil Prod Vol (bbl)_ Oil production is measured in barrels (bbl)
+  - _Gas Prod Vol (Mcf)_ Gas (natural gas) is measured in thousand cubic feet (mcf)
```
Comment (brentryanjohnson) [inline src/markdown/downloads/federal-production-by-month.md]: @lpgoldstein It looks like `mcf` is lowercase in the date file (unless I'm looking at the wrong file). We should probably case it the same here as in the data file.

File: src/markdown/downloads/federal-production.md [hunk 2]
```diff
@@ -68,35 +68,85 @@ We can only compare production that is reported in the same unit. The standard u
...
+  * _Muriate Of Potash-Granular_ (tons)
+  * _Muriate Of Potash-Standard_ (tons)
+  * _Oil_ (bbls)
+  * _Phosphate Raw Ore_ (tons)
+  * _Potash_ (tons)
+  * _Potassium Products_ (tons) include Manure Salts and Sylvite-Raw Ore
+  * _Potassium Sulphate-Standard_ (tons)
+  * _Purge Liquor_ (tons)
+  * _Quartz Crystal_ (tickets/lbs)
+  * _Salt_(tons)
```
Comment (brentryanjohnson) [inline src/markdown/downloads/federal-production.md]: @lpgoldstein Super minor (and probably my fault), but the "Salt" entry could use a space before "(tons)".

Comment (jennmalcolm) [reply src/markdown/downloads/federal-production.md]: I added a space. Are you ready to approve @brentryanjohnson?

**Pull request merged**